app/models/user.rb

```diff
@@ -1,11 +1,24 @@
 class User < ApplicationRecord
   devise :omniauthable, :timeoutable
+
   include Reauthable
 
+  before_create do
+    self.invitation_expires_at = Rails.configuration.x.auth.reauthenticate_in.from_now
```

Comment:
Should this use the `auth.invitation_ttl` config instead?

Reply:
@zheileman great catch! Thank you!